utils/evm_script.py

```diff
@@ -181,3 +152,40 @@ def get_error_msg(hash, values):
     hash = f"0x{keccak(text=error)[:4].hex()}"
     values = values if values else []
     return get_error_msg(hash, values)
+
+
+# https://eips.ethereum.org/EIPS/eip-55
+def checksum_encode(addr):  # Takes a 20-byte binary address as input
```

Comment:
Thank you for adding the checksum validation! :rocket: 

I wouldn't say these helpers are best suited to reside within `utils/evm_script.py` tbh :thinking: